src/dev/lupio/lupio_tty.cc

```diff
@@ -46,7 +46,8 @@ namespace gem5
 LupioTTY::LupioTTY(const Params &params)
     : BasicPioDevice(params, params.pio_size),
     writChar(-1), readChar(-1), writIntrEn(false), readIntrEn(false),
-    terminal(params.terminal), platform(params.platform)
+    terminal(params.terminal), pic(params.pic),
+    lupioTTYIntID(params.lupio_tty_int_id)
```

Comment:
```suggestion
    intID(params.int_id)
```

This is in the LupioTTY class. There's no need to repeat that in the variable name.